index.js

```diff
@@ -94,10 +99,12 @@ function send(ctx, path, opts) {
       throw err;
     }
 
+    if (setHeaders) setHeaders(ctx.res, path, stats);
```

Comment:
I mean:

```js
if (setHeaders) {
  function set(key, val) {
    if (!/(last-modified|cache-control)/i.test(key) {
      throw new TypeError('only support Last-Modified or Cache-Control');
    }
    return ctx.res.setHeader(key, val);
  }
  function get(key) {
    return ctx.res.getHeader(key);
  }
  setHeaders({set: set, get: get}, path, stats);
}
```

I don't think this will take too much /cc @coderhaoxin